src/main/java/Duke.java

```diff
@@ -1,10 +1,298 @@
+                listOfTasks[taskCount] = new Event(task, date);
+            }
+            break;
+        default:
+            listOfTasks[taskCount] = new Task(command);
+        }
+
+        // Informs user if their task is added succesfully
+        printAddResult(listOfTasks, taskCount, isDateRequired, isDateIncluded);
+    }
+
+    /**
+     * Prints the result of adding task (either successful or unsuccessful)
+     *
+     * @param listOfTasks Array containing tasks inserted by user
+     * @param taskCount Number of tasks inserted
+     * @param isDateRequired Determine if a date is required
+     * @param isDateIncluded Whether date is included for deadline/event
+     */
+    private static void printAddResult(Task[] listOfTasks, int taskCount, boolean isDateRequired, boolean isDateIncluded) {
+        // Task is added successfully when it does not require a date or if the date is included
+        if (!isDateRequired || isDateIncluded) {
+            printDivider();
+            System.out.println("Alrighty! I've added the following task:");
+            System.out.println(listOfTasks[taskCount]);
+            Task.printNumberOfTasks(); // Inform user how many tasks they have
+            printDivider();
+        } else {
+            botSpeak("Task not added. Please indicate a date using /by or /at followed by the date!");
+        }
+    }
+
+    /**
+     * Identifies and return the type of task the user has inserted
+     *
+     * @param command Command that the user input
+     * @return the type of the task (event, deadline, todo)
+     */
+    private static TaskType getTaskType(String command) {
+        TaskType taskType;
+
+        String commandModified = command.trim().toLowerCase();
+
+        // Check what is the type of the task given
+        if (commandModified.startsWith("todo")){
+            taskType = TaskType.TODO;
+        } else if (commandModified.startsWith("deadline")){
+            taskType = TaskType.DEADLINE;
+        } else if (commandModified.startsWith("event")){
+            taskType = TaskType.EVENT;
+        } else {
+            // taskType is NORMAL when user did not input specific type at the start
+            taskType = TaskType.NORMAL;
+        }
+        return taskType;
+    }
+
+    /**
+     * Prints out the list of tasks stored
+     *
+     * @param listOfTasks Array containing tasks inserted by user
+     */
+    public static void printList(Task[] listOfTasks){
+        int taskCount = Task.getNumberOfTasks();
+
+        // Notify the user if no tasks has been added yet
+        if (taskCount== 0) {
+            botSpeak("No tasks has been added yet. Try adding something!");
+        } else {
+            // Prints out the list of commands with respective index number
+            printDivider();
+            System.out.println("Here are the tasks in your list:");
+            for (int i=0 ; i < taskCount; i++){
```

Comment:
There is a space between int i=0 and the semi colon ;

Reply:
Aight thanks!